kernel/libc/include/stdlib.h

```diff
@@ -2,4 +2,8 @@
 #define _STDLIB_H_
 
 int abs(int);
-#endif
\ No newline at end of file
+void srand(int seed);
+int xorshift32();
```

Comment:
since rand and srand uses xorshift, do we need to put it on stdlib.h? Instead of can we put it on stdlib.c only and make it static? I don't know the how it't managed on stdlibc so this is just an idea, it's up to you :)